# request (none)
---
geometry_msgs/PoseStamped grasp_pose
float32 score
bool    success
string  message

